jellyfin_kodi/player.py

```diff
@@ -326,7 +326,11 @@ def report_playback(self, report=True):
         if not report:
 
             previous = item['CurrentPosition']
-            item['CurrentPosition'] = int(self.getTime())
+
+            try:
+                item['CurrentPosition'] = int(self.getTime())
+            except Exception:  # at this point we should be playing and if not then bail out
```

Comment:
We should at least log something when catching generic exceptions.
```suggestion
            except Exception as e:
                # getTime() raises RuntimeException if nothing is playing
                LOG.debug("Failed to get playback position: %s", e)
```

Reply:
Fair, I just copied this handling from elsewhere in this file, including the comment. I'm travelling atm so can't link it easily, but should be easy to find.

Reply:
That is entirely fair – we've been doing lots of cleanup in this addon since forking, but there's still a lot left, including generic exception catching